spec.emu

```diff
@@ -655,10 +656,14 @@ contributors: Google, Ecma International
             <li>
               <ins>A rangePatterns field with a Record value<ins>
                 <ul>
-                  <li><ins>The rangePatterns record may have any of the fields in <emu-xref href="#table-datetimeformat-rangepatternfields"></emu-xref>, where each field represents a range pattern and it's value is a list of Records.</ins>
+                  <li><ins>The rangePatterns record may have any of the fields in <emu-xref href="#table-datetimeformat-rangepatternfields"></emu-xref>, where each field represents a range pattern and it's value is a Record.</ins>
                     <ul>                  
                       <li><ins>The name of the field indicates the largest calendar element that must be different between the start and end dates in order to use this range pattern. For example, if the field name is Month, it contains the range pattern that should be used to format a date range where the Era and Year values are the same, but Month value is different.</ins></li>
-                      <li><ins>Each element of the list is a record representing a part of the range pattern. Each record contains a Pattern field and a Source field. The Pattern field's value is a String of the same format as the regular date pattern String. The Source field must contain one of the following values `"shared"`, `"startRange"` or `"endRange"` which indicates which of the range's dates should be formatted using the value of the Pattern field.</ins></li> 
+                      <li><ins>The record will contain the following fields:</ins></li>
+                        <ul>
+                          <li><ins>A subset of the fields shown in the Internal Slot column of <emu-xref href="#table-datetimeformat-components"></emu-xref>, where each field must have one of the values specified for that field in the Values column of <emu-xref href="#table-datetimeformat-components"></emu-xref>. Only the fields required to format a date with any of the PatternParts will be added.</ins></li>
+                          <li><ins>A PatternParts field whose value is a list of Records each representing a part of the range pattern. Each record contains a Pattern field and a Source field. The Pattern field's value is a String of the same format as the regular date pattern String. The Source field must contain one of the following values `"shared"`, `"startRange"` or `"endRange"` which indicates which of the range's dates should be formatted using the value of the Pattern field.</li></ins>
+                        </ul>
                     </ul>
                   </li>
                   <li><ins>The rangePatterns record must have a Default field which contains the default range pattern used when the specific range pattern is not available. It's value is a list of records with the same structure as the other fields in the rangePatterns record.</ins></li>                  
```

Comment:
Pre-existing: [[Default]] instead of plain "Default". And in the next list item, [[hour]], [[rangePatterns12]], etc.

Reply:
The reason why I used plain "Default" instead of [[Default]] (and others) is to be consistent with other paragraphs in this section of the spec which are doing the same. If we update this I'd like to update the pre-existing cases too. Do you think this should be updated as part of this PR/Proposal or should these issues be fixed in a follow-up spec-fix PR?

Reply:
I rebased latest changes in master. Now we are using [[Default]] instead of plain "Default" and others.